core/src/main/java/com/datastrato/gravitino/storage/relational/utils/POConverters.java

```diff
@@ -748,4 +779,82 @@ public static RolePO initializeRolePOWithVersion(RoleEntity roleEntity, RolePO.B
       throw new RuntimeException("Failed to serialize json object:", e);
     }
   }
+
+  /**
+   * Initialize GroupPO
+   *
+   * @param groupEntity GroupEntity object
+   * @return GroupPO object with version initialized
+   */
+  public static GroupPO initializeGroupPOWithVersion(
+      GroupEntity groupEntity, GroupPO.Builder builder) {
+    try {
+      return builder
+          .withGroupId(groupEntity.id())
+          .withGroupName(groupEntity.name())
+          .withAuditInfo(JsonUtils.anyFieldMapper().writeValueAsString(groupEntity.auditInfo()))
+          .withCurrentVersion(INIT_VERSION)
+          .withLastVersion(INIT_VERSION)
+          .withDeletedAt(DEFAULT_DELETED_AT)
+          .build();
+    } catch (JsonProcessingException e) {
+      throw new RuntimeException("Failed to serialize json object:", e);
+    }
+  }
+
+  /**
+   * Update GroupPO version
+   *
+   * @param oldGroupPO the old GroupPO object
+   * @param newGroup the new GroupEntity object
+   * @return GroupPO object with updated version
+   */
+  public static GroupPO updateGroupPOWithVersion(GroupPO oldGroupPO, GroupEntity newGroup) {
+    Long lastVersion = oldGroupPO.getLastVersion();
+    // Will set the version to the last version + 1 when having some fields need be multiple version
```

Comment:
This comment is confusing, I know that version value is useless here, but as you describe, you will increment it in the comment, but you won't do it accordingly in the code below.

Reply:
the version field is reserved and is not currently used, it same as other entity

Reply:
You should use
```
// TODO: set the version to the last version + 1 when having some fields need be multiple version
```